.buildkite/pipeline-for-tags.yml

```
@@ -32,3 +32,37 @@ steps:
    - export TEZOS_VERSION="$(cat nix/nix/sources.json | jq -r '.tezos.ref' | cut -d'/' -f3)"
    - buildkite-agent artifact download "out/*" . --step build-source-packages
    - ./scripts/publish-native-packages.sh out
+
+ - label: Build Big Sur x86_64 bottles
+   key: build-bottles-big-sur-x86_64
+   if: build.tag =~ /^v.*/
+   agents:
+     queue: "x86_64-rosetta-darwin"
+   commands:
+   - nix-shell ./scripts/shell.nix --run './scripts/build-all-bottles.sh "big_sur"'
+   artifact_paths:
+     - '*.bottle.*'
+
```

```suggestion

 - wait

```

Brew may misbehave when there are two `brew` processes running together